cypress.json

```diff
@@ -1 +1,4 @@
-{}
+{
+    "integrationFolder": "cypress/automated-tests",
+    "defaultCommandTimeout": 10000
```

Comment:
Default Cypress timeout is 4000ms, which sometimes causes test flakiness during file uploads and final submission. The script will wait up to 10000ms for actions, but progress sooner if the action completes before timeout.